lib/pengine/unpack.c

```diff
@@ -1578,20 +1583,20 @@ determine_online_status(xmlNode * node_state, pe_node_t * this_node, pe_working_
     }
 
     if (this_node->details->type == node_ping) {
-        crm_info("Node %s is not a Pacemaker node", this_node->details->uname);
+        crm_info("%s is not a Pacemaker node", pe__node_name(this_node));
 
     } else if (this_node->details->unclean) {
-        pe_proc_warn("Node %s is unclean", this_node->details->uname);
+        pe_proc_warn("Node %s is unclean", pe__node_name(this_node));
```

Comment:
You could get rid of the leading `"Node "` in this string and it wouldn't be any more weird looking than the ones in the next two branches.

Reply:
Sure